app/controllers/hosts_controller.rb

```diff
@@ -193,9 +193,6 @@ def externalNodes
 
     begin
       respond_to do |format|
-        # don't break lines in yaml to support Ruby < 1.9.3
-        # Remove the HashesWithIndifferentAccess using 'deep_stringify_keys',
-        # then we turn it into YAML
```

Comment:
This part is still relevant

Reply:
It is but we don't put comments on every line explaining what it does, do we? ;-)

I felt like it's not needed.

Reply:
Well, you yourself in this PR missed why it was useful so clearly it's not obvious.